diff --git a/browser/components/urlbar/UrlbarProviderSearchSuggestions.sys.mjs b/browser/components/urlbar/UrlbarProviderSearchSuggestions.sys.mjs
index 90c39d04e1..5f293ade6e 100644
--- a/browser/components/urlbar/UrlbarProviderSearchSuggestions.sys.mjs
+++ b/browser/components/urlbar/UrlbarProviderSearchSuggestions.sys.mjs
@@ -23,7 +23,7 @@ ChromeUtils.defineESModuleGetters(lazy, {
   UrlbarPrefs: "moz-src:///browser/components/urlbar/UrlbarPrefs.sys.mjs",
   UrlbarProviderTopSites:
     "moz-src:///browser/components/urlbar/UrlbarProviderTopSites.sys.mjs",
-  UrlbarResult: "chrome://browser/content/urlbar/UrlbarResult.mjs",
+  UrlbarResult: "moz-src:///browser/components/urlbar/UrlbarResult.sys.mjs",
   UrlbarShared: "chrome://browser/content/urlbar/UrlbarShared.mjs",
   UrlbarSearchUtils:
     "moz-src:///browser/components/urlbar/UrlbarSearchUtils.sys.mjs",
@@ -96,9 +96,9 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
     // If the sources don't include search or the user used a restriction
     // character other than search, don't allow any suggestions.
     if (
-      !queryContext.sources.includes(lazy.UrlbarShared.RESULT_SOURCE.SEARCH) ||
+      !queryContext.sources.includes(UrlbarUtils.RESULT_SOURCE.SEARCH) ||
       (queryContext.restrictSource &&
-        queryContext.restrictSource != lazy.UrlbarShared.RESULT_SOURCE.SEARCH)
+        queryContext.restrictSource != UrlbarUtils.RESULT_SOURCE.SEARCH)
     ) {
       return false;
     }
@@ -139,13 +139,12 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
     return (
       queryContext.searchString.startsWith("@") ||
       (queryContext.restrictSource &&
-        queryContext.restrictSource ==
-          lazy.UrlbarShared.RESULT_SOURCE.SEARCH) ||
+        queryContext.restrictSource == UrlbarUtils.RESULT_SOURCE.SEARCH) ||
       queryContext.tokens.some(
-        t => t.type == lazy.UrlbarShared.TOKEN_TYPE.RESTRICT_SEARCH
+        t => t.type == lazy.UrlbarTokenizer.TYPE.RESTRICT_SEARCH
       ) ||
       (queryContext.searchMode &&
-        queryContext.sources.includes(lazy.UrlbarShared.RESULT_SOURCE.SEARCH))
+        queryContext.sources.includes(UrlbarUtils.RESULT_SOURCE.SEARCH))
     );
   }
 
@@ -236,7 +235,7 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
    * @param {UrlbarQueryContext} queryContext
    * @param {(provider: UrlbarProvider, result: UrlbarResult) => void} addCallback
    *   Callback invoked by the provider to add a new result.
-   * @param {UrlbarParentController} controller The UrlbarParentController instance.
+   * @param {UrlbarController} controller The UrlbarController instance.
    */
   async startQuery(queryContext, addCallback, controller) {
     let instance = this.queryInstance;
@@ -266,7 +265,7 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
       lazy.UrlbarTokenizer.isRestrictionToken(queryContext.tokens[0]) &&
       (queryContext.tokens.length > 1 ||
         queryContext.tokens[0].type ==
-          lazy.UrlbarShared.TOKEN_TYPE.RESTRICT_SEARCH)
+          lazy.UrlbarTokenizer.TYPE.RESTRICT_SEARCH)
     ) {
       leadingRestrictionToken = queryContext.tokens[0].value;
     }
@@ -275,7 +274,7 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
     // when the search shortcut is used and it's not user typed. Don't strip
     // other restriction chars, so that it's possible to search for things
     // including one of those (e.g. "c#").
-    if (leadingRestrictionToken === lazy.UrlbarShared.RESTRICT_TOKENS.SEARCH) {
+    if (leadingRestrictionToken === lazy.UrlbarTokenizer.RESTRICT.SEARCH) {
       query = UrlbarUtils.substringAfter(query, leadingRestrictionToken).trim();
     }
 
@@ -346,14 +345,14 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
       return /** @type {UrlbarResultCommand[]} */ ([
         {
           name: RESULT_MENU_COMMANDS.TRENDING_BLOCK,
-          l10n: { id: "urlbar-result-menu-trending-dont-show2" },
+          l10n: { id: "urlbar-result-menu-trending-dont-show" },
         },
         {
           name: "separator",
         },
         {
           name: RESULT_MENU_COMMANDS.TRENDING_HELP,
-          l10n: { id: "urlbar-result-menu-learn-more2" },
+          l10n: { id: "urlbar-result-menu-learn-more" },
         },
       ]);
     }
@@ -531,8 +530,8 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
 
         results.push(
           new lazy.UrlbarResult({
-            type: lazy.UrlbarShared.RESULT_TYPE.SEARCH,
-            source: lazy.UrlbarShared.RESULT_SOURCE.SEARCH,
+            type: UrlbarUtils.RESULT_TYPE.SEARCH,
+            source: UrlbarUtils.RESULT_SOURCE.SEARCH,
             isRichSuggestion: !!entry.icon,
             richSuggestionIconSize: entry.icon
               ? UrlbarProviderSearchSuggestions.RICH_ICON_SIZE
@@ -642,13 +641,7 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
    *   Whether we should fetch trending results.
    */
   #shouldFetchTrending(queryContext) {
-    return !!(
-      queryContext.searchString == "" &&
-      lazy.UrlbarPrefs.get("trending.featureGate") &&
-      lazy.UrlbarPrefs.get("suggest.trending") &&
-      (queryContext.searchMode ||
-        !lazy.UrlbarPrefs.get("trending.requireSearchMode"))
-    );
+    return false;
   }
 
   /*
@@ -687,15 +680,15 @@ export class UrlbarProviderSearchSuggestions extends UrlbarProvider {
 
 function makeFormHistoryResult(queryContext, engine, entry) {
   return new lazy.UrlbarResult({
-    type: lazy.UrlbarShared.RESULT_TYPE.SEARCH,
-    source: lazy.UrlbarShared.RESULT_SOURCE.HISTORY,
+    type: UrlbarUtils.RESULT_TYPE.SEARCH,
+    source: UrlbarUtils.RESULT_SOURCE.HISTORY,
     payload: {
       engine: engine.name,
       suggestion: entry.value,
       title: entry.value,
       lowerCaseSuggestion: entry.value.toLocaleLowerCase(),
       isBlockable: true,
-      blockL10n: { id: "urlbar-result-menu-remove-from-history2" },
+      blockL10n: { id: "urlbar-result-menu-remove-from-history" },
       helpUrl:
         Services.urlFormatter.formatURLPref("app.support.baseURL") +
         "awesome-bar-result-menu",
