--- a/browser/locales/en-US/browser/preonboarding.ftl
+++ b/browser/locales/en-US/browser/preonboarding.ftl
@@ -1,17 +1,15 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 ## Preonboarding modal screen strings
-
 preonboarding-title = Welcome to { -brand-product-name }
-preonboarding-subtitle = By continuing, you agree to the { -brand-product-name } Terms of Use and our Privacy Notice. To help improve the browser, { -brand-product-name } sends diagnostic and interaction data to { -vendor-short-name }.
+preonboarding-subtitle = By continuing, you agree to the { -brand-product-name } Terms of Use and our Privacy Notice. To help improve the Gorilla browser, { -brand-product-name } sends diagnostic and interaction data to { -vendor-short-name }.
 preonboarding-primary-cta = Confirm and continue
 preonboarding-primary-cta-v2 = Continue
 
 # Labels for header style button that toggles showing the terms of use, privacy notice, and data preferences checklist
-preonboarding-manage-and-read-header = Manage data collection settings and read our Terms of Use and Privacy Notice
-preonboarding-manage-and-read-header-v2 = Read Terms of Use and Privacy Notice. Manage additional settings.
+preonboarding-manage-and-read-header = Manage data collection Gorilla settings and read our Terms of Use and Privacy Notice
+preonboarding-manage-and-read-header-v2 = Read Terms of Use and Privacy Notice. Manage additional Gorilla settings.
 
 # Label for header style button that toggles showing the Terms of Use
 preonboarding-terms-of-use-header-button-title = Read our Terms of Use
@@ -33,4 +31,4 @@
 preonboarding-checklist-interaction-data-label = Send technical and interaction data to { -vendor-short-name }
 preonboarding-checklist-interaction-data-description = Data about your device, hardware configuration, and how you use { -brand-product-name } helps improve features, performance, and stability for users everywhere.
 preonboarding-checklist-crash-reports-label = Automatically send crash reports
-preonboarding-checklist-crash-reports-description = Crash reports allow us to diagnose and fix issues with the browser. Reports may include personal or sensitive data.
+preonboarding-checklist-crash-reports-description = Crash reports allow us to diagnose and fix issues with the Gorilla browser. Reports may include personal or sensitive data.
