--- a/browser/locales/en-US/browser/policy-messages.ftl
+++ b/browser/locales/en-US/browser/policy-messages.ftl
@@ -1,6 +1,5 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-compulsory-restart-message = <strong>A new version of { -brand-short-name } is available.</strong> Your administrator will restart your browser at { DATETIME($datetime, dateStyle: "short", timeStyle: "short") }. Tabs will reopen.
+compulsory-restart-message = <strong>A new version of { -brand-short-name } is available.</strong> Your administrator will restart your Gorilla browser at { DATETIME($datetime, dateStyle: "short", timeStyle: "short") }. Gorilla Tabs will reopen.
 policy-update-now = Update now
