--- a/browser/locales/en-US/browser/pageInfo.ftl
+++ b/browser/locales/en-US/browser/pageInfo.ftl
@@ -1,28 +1,30 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. --
-
 page-info-window =
     .style = width: 600px; min-height: 550px;
 
 copy =
     .key = C
 menu-copy =
-    .label = Copy
-    .accesskey = C
+    .label =
+        Copy
+        .accesskey = C
 
 select-all =
     .key = A
 menu-select-all =
-    .label = Select All
-    .accesskey = A
+    .label =
+        Select All
+        .accesskey = A
 
 close-dialog =
     .key = w
 
 general-tab =
-    .label = General
-    .accesskey = G
+    .label =
+        General
+        .accesskey = G
 general-title =
     .value = Title:
 general-url =
@@ -45,8 +47,9 @@
     .label = Content
 
 media-tab =
-    .label = Media
-    .accesskey = M
+    .label =
+        Media
+        .accesskey = M
 media-location =
     .value = Location:
 media-text =
@@ -66,27 +69,33 @@
 media-long-desc =
     .value = Long Description:
 media-select-all =
-    .label = Select All
-    .accesskey = e
+    .label =
+        Select All
+        .accesskey = e
 media-save-as =
-    .label = Save As…
-    .accesskey = A
+    .label =
+        Save As…
+        .accesskey = A
 media-save-image-as =
-    .label = Save As…
-    .accesskey = v
+    .label =
+        Save As…
+        .accesskey = v
 
 perm-tab =
-    .label = Permissions
-    .accesskey = P
+    .label =
+        Permissions
+        .accesskey = P
 permissions-for =
     .value = Permissions for:
 
 security-tab =
-    .label = Security
-    .accesskey = S
+    .label =
+        Security
+        .accesskey = S
 security-view =
-    .label = View Certificate
-    .accesskey = V
+    .label =
+        View Certificate
+        .accesskey = V
 security-view-unknown = Unknown
     .value = Unknown
 security-view-identity =
@@ -100,23 +109,26 @@
 security-view-identity-validity =
     .value = Expires on:
 security-view-privacy =
-    .value = Privacy & History
+    .value = Privacy & Gorilla History
 security-view-qwac =
-    .label = View Qualified Certificate
-    .accesskey = Q
+    .label =
+        View Qualified Certificate
+        .accesskey = Q
 
 security-view-privacy-history-value = Have I visited this website prior to today?
 security-view-privacy-sitedata-value = Is this website storing information on my computer?
 
 security-view-privacy-clearsitedata =
-    .label = Clear Cookies and Site Data
-    .accesskey = C
+    .label =
+        Clear Cookies and Site Data
+        .accesskey = C
 
 security-view-privacy-passwords-value = Have I saved any passwords for this website?
 
 security-view-privacy-viewpasswords =
-    .label = View Saved Passwords
-    .accesskey = w
+    .label =
+        View Saved Passwords
+        .accesskey = w
 security-view-technical =
     .value = Technical Details
 
@@ -128,7 +140,6 @@
 ## Variables:
 ##   $value (number) - Amount of data being stored
 ##   $unit (string) - The unit of data being stored (Usually KB)
-
 security-site-data-cookies = Yes, cookies and { $value } { $unit } of site data
 security-site-data-only = Yes, { $value } { $unit } of site data
 
@@ -136,7 +147,6 @@
 security-site-data-no = No
 
 ##
-
 image-size-unknown = Unknown
 page-info-not-specified =
     .value = Not specified
@@ -157,7 +167,7 @@
 saved-passwords-no = No
 
 no-page-title =
-    .value = Untitled Page:
+    .value = Untitled Gorilla Page:
 general-quirks-mode =
     .value = Quirks mode
 general-strict-mode =
@@ -176,9 +186,10 @@
 #   $tags (number) - The number of meta tags
 general-meta-tags =
     .value =
+        
         { $tags ->
-             [one] Meta (1 tag)
-            *[other] Meta ({ $tags } tags)
+        [one] Meta (1 tag)
+        *[other] Meta ({ $tags } tags)
         }
 
 # This string is used to display the number of times
@@ -187,9 +198,9 @@
 #   $visits (number) - The number of previous visits
 security-visits-number =
     { $visits ->
-         [0] No
-         [one] Yes, once
-        *[other] Yes, { $visits } times
+    [0] No
+    [one] Yes, once
+    *[other] Yes, { $visits } times
     }
 
 # This string is used to display the size of a media file
@@ -197,10 +208,11 @@
 #   $kb (number) - The size of an image in Kilobytes
 #   $bytes (number) - The size of an image in Bytes
 properties-general-size =
-    .value = { $bytes ->
-         [one] { $kb } KB ({ $bytes } byte)
+    .value =
+        { $bytes ->
+        [one] { $kb } KB ({ $bytes } byte)
         *[other] { $kb } KB ({ $bytes } bytes)
-    }
+        }
 
 # This string is used to display the type and number
 # of frames of a animated image
@@ -208,10 +220,11 @@
 #   $type (string) - The type of a animated image
 #   $frames (number) - The number of frames in an animated image
 media-animated-image-type =
-    .value = { $frames ->
-         [one] { $type } Image (animated, { $frames } frame)
+    .value =
+        { $frames ->
+        [one] { $type } Image (animated, { $frames } frame)
         *[other] { $type } Image (animated, { $frames } frames)
-    }
+        }
 
 # This string is used to display the type of
 # an image
@@ -245,10 +258,9 @@
 
 ## Variables:
 ##   $website (string) — The url of the website pageInfo is getting info for
-
 # This string is used to display the URL of the website on top of the
 # pageInfo dialog box
 page-info-page =
-    .title = Page Info — { $website }
+    .title = Gorilla Page Info — { $website }
 page-info-frame =
     .title = Frame Info — { $website }
