--- a/browser/locales/en-US/browser/sidebarMenu.ftl
+++ b/browser/locales/en-US/browser/sidebarMenu.ftl
@@ -1,20 +1,19 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 sidebar-menu-bookmarks =
-  .label = Bookmarks
+    .label = Gorilla Bookmarks
 
-sidebar-menu-bookmarks-2 = Bookmarks
+sidebar-menu-bookmarks-2 = Gorilla Bookmarks
 
 sidebar-menu-history =
-  .label = History
+    .label = Gorilla History
 
 sidebar-menu-synced-tabs =
-  .label = Synced Tabs
+    .label = Synced Gorilla Tabs
 
 sidebar-menu-close =
-  .label = Close Sidebar
+    .label = Close Gorilla Sidebar
 
 sidebar-close-button =
-  .tooltiptext = Close sidebar
+    .tooltiptext = Close Gorilla sidebar
