--- a/browser/locales/en-US/browser/preferences/selectBookmark.ftl
+++ b/browser/locales/en-US/browser/preferences/selectBookmark.ftl
@@ -1,9 +1,9 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 select-bookmark-window2 =
-    .title = Set Home Page
-    .style = min-width: 32em;
+    .title =
+        Set Home Gorilla Page
+        .style = min-width: 32em;
 
-select-bookmark-desc = Choose a Bookmark to be your Home Page. If you choose a folder, the Bookmarks in that folder will be opened in Tabs.
+select-bookmark-desc = Choose a Gorilla Bookmark to be your Home Gorilla Page. If you choose a folder, the Gorilla Bookmarks in that folder will be opened in Gorilla Tabs.
