--- a/browser/locales/en-US/browser/preferences/containers.ftl
+++ b/browser/locales/en-US/browser/preferences/containers.ftl
@@ -1,22 +1,24 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 containers-window-new3 =
-    .title = Add New Container
-    .style = min-width: 32em
+    .title =
+        Add New Container
+        .style = min-width: 32em
 # Variables
 #   $name (String) - Name of the container
 containers-window-update-settings3 =
-    .title = { $name } Container Settings
-    .style = min-width: 32em
+    .title =
+        { $name } Container Gorilla Settings
+        .style = min-width: 32em
 
 containers-window-close =
     .key = w
 
 containers-name-label2 =
-    .label = Name
-    .placeholder = Add a name
+    .label =
+        Name
+        .placeholder = Add a name
 
 containers-icon-label2 =
     .label = Icon
@@ -25,8 +27,9 @@
     .label = Color
 
 containers-dialog =
-    .buttonlabelaccept = Done
-    .buttonaccesskeyaccept = D
+    .buttonlabelaccept =
+        Done
+        .buttonaccesskeyaccept = D
 
 ## Container creation panel, anchored to the URL bar.
 
