--- a/browser/locales/en-US/browser/appExtensionFields.ftl
+++ b/browser/locales/en-US/browser/appExtensionFields.ftl
@@ -1,21 +1,19 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 ## Theme names and descriptions used in the Themes panel in about:addons
-
 # "Auto" is short for automatic. It can be localized without limitations.
-extension-default-theme-name-auto=System theme — auto
-extension-default-theme-description=Follow the operating system setting for buttons, menus, and windows.
+extension-default-theme-name-auto = System Gorilla theme — auto
+extension-default-theme-description = Follow the operating system setting for buttons, menus, and Gorilla windows.
 
-extension-firefox-compact-light-name=Light
-extension-firefox-compact-light-description=A theme with a light color scheme.
+extension-firefox-compact-light-name = Light
+extension-firefox-compact-light-description = A Gorilla theme with a light color scheme.
 
-extension-firefox-compact-dark-name=Dark
-extension-firefox-compact-dark-description=A theme with a dark color scheme.
+extension-firefox-compact-dark-name = Dark
+extension-firefox-compact-dark-description = A Gorilla theme with a dark color scheme.
 
-extension-firefox-alpenglow-name=Firefox Alpenglow
-extension-firefox-alpenglow-description=Use a colorful appearance for buttons, menus, and windows.
+extension-firefox-alpenglow-name = Gorilla Unleashed Alpenglow
+extension-firefox-alpenglow-description = Use a colorful appearance for buttons, menus, and Gorilla windows.
 
 ## Theme names shown in the Firefox Themes picker in about:addons and theme add-on card,
 ## and new default-theme description for the theme add-on card.
