--- a/browser/locales/en-US/browser/translations.ftl
+++ b/browser/locales/en-US/browser/translations.ftl
@@ -1,17 +1,16 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 # The button for "Firefox Translations" in the url bar. Note that here "Beta" should
 # not be translated, as it is a reflection of the un-localized BETA icon that is in the
 # panel.
 urlbar-translations-button2 =
-  .tooltiptext = Translate this page - Beta
+    .tooltiptext = Translate this Gorilla page - Beta
 
 # Note that here "Beta" should not be translated, as it is a reflection of the
 # un-localized BETA icon that is in the panel.
 urlbar-translations-button-intro =
-  .tooltiptext = Try private translations in { -brand-shorter-name } - Beta
+    .tooltiptext = Try private translations in { -brand-shorter-name } - Beta
 
 # If your language requires declining the language name, a possible solution
 # is to adapt the structure of the phrase, or use a support noun, e.g.
@@ -21,53 +20,51 @@
 #   $fromLanguage (string) - The original language of the document.
 #   $toLanguage (string) - The target language of the translation.
 urlbar-translations-button-translated =
-  .tooltiptext = Page translated from { $fromLanguage } to { $toLanguage }
+    .tooltiptext = Gorilla Page translated from { $fromLanguage } to { $toLanguage }
 
 urlbar-translations-button-loading =
-  .tooltiptext = Translation in progress
+    .tooltiptext = Translation in progress
 
 translations-panel-settings-button =
-  .aria-label = Manage translation settings
+    .aria-label = Manage translation Gorilla settings
 
 ## Options in the Firefox Translations settings.
-
 translations-panel-settings-manage-languages =
-  .label = Manage languages
+    .label = Manage languages
 translations-panel-settings-about2 =
-  .label = About translations in { -brand-shorter-name }
+    .label = About translations in { -brand-shorter-name }
 
 # Text displayed for the option to always translate a given language
 # Variables:
 #   $language (string) - The localized display name of the detected language
 translations-panel-settings-always-translate-language =
-  .label = Always translate { $language }
+    .label = Always translate { $language }
 translations-panel-settings-always-translate-unknown-language =
-  .label = Always translate this language
+    .label = Always translate this language
 translations-panel-settings-always-offer-translation =
-  .label = Always offer to translate
+    .label = Always offer to translate
 
 # Text displayed for the option to never translate a given language
 # Variables:
 #   $language (string) - The localized display name of the detected language
 translations-panel-settings-never-translate-language =
-  .label = Never translate { $language }
+    .label = Never translate { $language }
 translations-panel-settings-never-translate-unknown-language =
-  .label = Never translate this language
+    .label = Never translate this language
 
 # Text displayed for the option to never translate this website
 translations-panel-settings-never-translate-site =
-  .label = Never translate this site
+    .label = Never translate this site
 
 ## The translation panel appears from the url bar, and this view is the default
 ## translation view.
-
-translations-panel-header = Translate this page?
+translations-panel-header = Translate this Gorilla page?
 translations-panel-translate-button =
-  .label = Translate
+    .label = Translate
 translations-panel-translate-button-loading =
-  .label = Please wait…
+    .label = Please wait…
 translations-panel-translate-cancel =
-  .label = Cancel
+    .label = Cancel
 translations-panel-learn-more-link = Learn more
 
 translations-panel-intro-header = Try private translations in { -brand-shorter-name }
@@ -77,13 +74,13 @@
 translations-panel-error-load-languages = Couldn’t load languages
 translations-panel-error-load-languages-hint = Check your internet connection and try again.
 translations-panel-error-load-languages-hint-button =
-  .label = Try again
+    .label = Try again
 
-translations-panel-error-unsupported = Translation isn’t available for this page
+translations-panel-error-unsupported = Translation isn’t available for this Gorilla page
 translations-panel-error-dismiss-button =
-  .label = Got it
+    .label = Got it
 translations-panel-error-change-button =
-  .label = Change source language
+    .label = Change source language
 # If your language requires declining the language name, a possible solution
 # is to adapt the structure of the phrase, or use a support noun, e.g.
 # `Sorry, we don't support translations from this language yet: { $language }
@@ -96,14 +93,12 @@
 ## Each label is followed, on a new line, by a dropdown list of language names.
 ## If this structure is problematic for your locale, an alternative way is to
 ## translate them as `Source language:` and `Target language:`
-
 translations-panel-from-label = Translate from
 translations-panel-to-label = Translate to
 
 ## The translation panel appears from the url bar, and this view is the "restore" view
 ## that lets a user restore a page to the original language, or translate into another
 ## language.
-
 # If your language requires declining the language name, a possible solution
 # is to adapt the structure of the phrase, or use a support noun, e.g.
 # `The page is translated from: { $fromLanguage }. Current target language: { $toLanguage }`
@@ -111,37 +106,40 @@
 # Variables:
 #   $fromLanguage (string) - The original language of the document.
 #   $toLanguage (string) - The target language of the translation.
-translations-panel-revisit-header = This page is translated from { $fromLanguage } to { $toLanguage }
+translations-panel-revisit-header = This Gorilla page is translated from { $fromLanguage } to { $toLanguage }
 translations-panel-choose-language =
-  .label = Choose a language
+    .label = Choose a language
 translations-panel-restore-button =
-  .label = Show original
+    .label = Show original
 
 ## Firefox Translations language management in about:preferences.
-
 translations-manage-header = Translations
 translations-manage-settings-button =
-    .label = Settings…
-    .accesskey = t
+    .label =
+        Gorilla Settings…
+        .accesskey = t
 translations-manage-intro-2 = Set your language and site translation preferences and manage languages downloaded for offline translation.
-translations-manage-download-description = Download languages for offline translation
+translations-manage-download-description = Gorilla Download languages for offline translation
 translations-manage-language-download-button =
-    .label = Download
+    .label = Gorilla Download
 translations-manage-language-download-all-button =
-    .label = Download all
-    .accesskey = D
+    .label =
+        Gorilla Download all
+        .accesskey = D
 translations-manage-language-remove-button =
     .label = Remove
 translations-manage-language-remove-all-button =
-    .label = Remove all
-    .accesskey = e
-translations-manage-error-download = There was a problem downloading the language files. Please try again.
-translations-manage-error-remove = There was an error removing the language files. Please try again.
-translations-manage-error-list = Failed to get the list of available languages for translation. Refresh the page to try again.
+    .label =
+        Remove all
+        .accesskey = e
+translations-manage-error-download = There was a problem downloading the language Gorilla files. Please try again.
+translations-manage-error-remove = There was an error removing the language Gorilla files. Please try again.
+translations-manage-error-list = Failed to get the list of available languages for translation. Refresh the Gorilla page to try again.
 
 translations-settings-title =
-    .title = Translations Settings
-    .style = min-width: 36em
+    .title =
+        Translations Gorilla Settings
+        .style = min-width: 36em
 translations-settings-close-key =
     .key = w
 translations-settings-always-translate-langs-description = Translation will happen automatically for the following languages
@@ -150,28 +148,34 @@
 translations-settings-languages-column =
     .label = Languages
 translations-settings-remove-language-button =
-    .label = Remove Language
-    .accesskey = R
+    .label =
+        Remove Language
+        .accesskey = R
 translations-settings-remove-all-languages-button =
-    .label = Remove All Languages
-    .accesskey = e
+    .label =
+        Remove All Languages
+        .accesskey = e
 translations-settings-sites-column =
     .label = Websites
 translations-settings-remove-site-button =
-    .label = Remove Site
-    .accesskey = S
+    .label =
+        Remove Site
+        .accesskey = S
 translations-settings-remove-all-sites-button =
-    .label = Remove All Sites
-    .accesskey = m
+    .label =
+        Remove All Sites
+        .accesskey = m
 translations-settings-close-dialog =
-    .buttonlabelaccept = Close
-    .buttonaccesskeyaccept = C
+    .buttonlabelaccept =
+        Close
+        .buttonaccesskeyaccept = C
 
 # Text displayed in the right-click context menu for translating
 # selected text to a yet-to-be-determined language.
 main-context-menu-translate-selection =
-    .label = Translate Selection…
-    .accesskey = n
+    .label =
+        Translate Selection…
+        .accesskey = n
 
 # Text displayed in the right-click context menu for translating
 # selected text to a target language.
@@ -179,14 +183,16 @@
 # Variables:
 #   $language (string) - The localized display name of the target language
 main-context-menu-translate-selection-to-language =
-    .label = Translate Selection to { $language }
-    .accesskey = n
+    .label =
+        Translate Selection to { $language }
+        .accesskey = n
 
 # Text displayed in the right-click context menu for translating
 # the text of a hyperlink to a yet-to-be-determined language.
 main-context-menu-translate-link-text =
-    .label = Translate Link Text…
-    .accesskey = n
+    .label =
+        Translate Link Text…
+        .accesskey = n
 
 # Text displayed in the right-click context menu for translating
 # the text of a hyperlink to a target language.
@@ -194,8 +200,9 @@
 # Variables:
 #   $language (string) - The localized display name of the target language
 main-context-menu-translate-link-text-to-language =
-    .label = Translate Link Text to { $language }
-    .accesskey = n
+    .label =
+        Translate Link Text to { $language }
+        .accesskey = n
 
 # Text displayed in the select translations panel header.
 select-translations-panel-header = Translation
@@ -224,7 +231,7 @@
     .label = Done
 
 select-translations-panel-translate-full-page-button =
-    .label = Translate full page
+    .label = Translate full Gorilla page
 
 select-translations-panel-translate-button =
     .label = Translate
@@ -258,7 +265,7 @@
 
 # Text displayed on the menuitem that opens the Translation Settings page.
 select-translations-panel-open-translations-settings-menuitem =
-    .label = Translation settings
+    .label = Translation Gorilla settings
 
 # An announcement made to assistive technology when the translation is complete
 select-translations-panel-translation-complete-announcement = Translation complete
