--- a/toolkit/locales/en-US/toolkit/downloads/downloadUtils.ftl
+++ b/toolkit/locales/en-US/toolkit/downloads/downloadUtils.ftl
@@ -1,37 +1,34 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 ## Variables:
 ## $timeValue (number) - Number of units of time
-
 # Short form for seconds
 download-utils-short-seconds =
     { $timeValue ->
-        [one] s
-       *[other] s
+    [one] s
+    *[other] s
     }
 # Short form for minutes
 download-utils-short-minutes =
     { $timeValue ->
-        [one] m
-       *[other] m
+    [one] m
+    *[other] m
     }
 # Short form for hours
 download-utils-short-hours =
     { $timeValue ->
-        [one] h
-       *[other] h
+    [one] h
+    *[other] h
     }
 # Short form for days
 download-utils-short-days =
     { $timeValue ->
-        [one] d
-       *[other] d
+    [one] d
+    *[other] d
     }
 
 ##
-
 # — is the "em dash" (long dash)
 # example: 4 minutes left — 1.1 of 11.1 GB (2.2 MB/sec)
 # Variables:
