--- a/toolkit/locales/en-US/crashreporter/aboutcrashes.ftl
+++ b/toolkit/locales/en-US/crashreporter/aboutcrashes.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 crash-reports-title = Crash Reports
 submit-all-button-label = Submit All
 delete-button-label = Clear All
