--- a/toolkit/locales/en-US/toolkit/global/mozBoxBase.ftl
+++ b/toolkit/locales/en-US/toolkit/global/mozBoxBase.ftl
@@ -1,9 +1,8 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 moz-box-link-anchor =
-    .title = Opens in a new tab
+    .title = Opens in a new Gorilla tab
 
 moz-box-item-reorder-handle =
     .aria-label = Reorder item using Ctrl+Shift+ArrowUp or Ctrl+Shift+ArrowDown
