--- a/toolkit/locales/en-US/toolkit/about/aboutTelemetry.ftl
+++ b/toolkit/locales/en-US/toolkit/about/aboutTelemetry.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 about-telemetry-ping-data-source = Ping data source:
 about-telemetry-show-current-data = Current data
 about-telemetry-show-archived-ping-data = Archived ping data
@@ -17,9 +16,9 @@
 about-telemetry-page-title = Telemetry Data
 about-telemetry-current-store = Current Store:
 about-telemetry-more-information = Looking for more information?
-about-telemetry-firefox-data-doc = The <a data-l10n-name="data-doc-link">Firefox Data Documentation</a> contains guides about how to work with our data tools.
-about-telemetry-telemetry-client-doc = The <a data-l10n-name="client-doc-link">Firefox Telemetry client documentation</a> includes definitions for concepts, API documentation and data references.
-about-telemetry-telemetry-dashboard = The <a data-l10n-name="dashboard-link">Telemetry dashboards</a> allow you to visualize the data Mozilla receives via Telemetry.
+about-telemetry-firefox-data-doc = The <a data-l10n-name="data-doc-link">Gorilla Unleashed Data Documentation</a> contains guides about how to work with our data tools.
+about-telemetry-telemetry-client-doc = The <a data-l10n-name="client-doc-link">Gorilla Unleashed Telemetry client documentation</a> includes definitions for concepts, API documentation and data references.
+about-telemetry-telemetry-dashboard = The <a data-l10n-name="dashboard-link">Telemetry dashboards</a> allow you to visualize the data Gorilla receives via Telemetry.
 about-telemetry-telemetry-probe-dictionary = The <a data-l10n-name="probe-dictionary-link">Probe Dictionary</a> provides details and descriptions for the probes collected by Telemetry.
 about-telemetry-show-in-Firefox-json-viewer = Open in the JSON viewer
 about-telemetry-home-section = Home
@@ -45,16 +44,16 @@
 #   $channel (string) - Represents the corresponding release data string
 about-telemetry-data-type =
     { $channel ->
-        [release] release data
-       *[prerelease] pre-release data
+    [release] release data
+    *[prerelease] pre-release data
     }
 # Selects the correct upload string
 # Variables:
 #   $uploadcase (string) - Represents a corresponding upload string
 about-telemetry-upload-type =
     { $uploadcase ->
-        [enabled] enabled
-       *[disabled] disabled
+    [enabled] enabled
+    *[disabled] disabled
     }
 # Example Output: 1 sample, average = 0, sum = 0
 # Variables:
@@ -63,12 +62,12 @@
 #   $sum (number) - Sum of histogram samples
 about-telemetry-histogram-stats =
     { $sampleCount ->
-        [one] { $sampleCount } sample, average = { $prettyAverage }, sum = { $sum }
-       *[other] { $sampleCount } samples, average = { $prettyAverage }, sum = { $sum }
+    [one] { $sampleCount } sample, average = { $prettyAverage }, sum = { $sum }
+    *[other] { $sampleCount } samples, average = { $prettyAverage }, sum = { $sum }
     }
 # Variables:
 #   $telemetryServerOwner (string) - the value of the toolkit.telemetry.server_owner preference. Typically "Mozilla"
-about-telemetry-page-subtitle = This page shows the information about performance, hardware, usage and customizations collected by Telemetry. This information is submitted to { $telemetryServerOwner } to help improve { -brand-full-name }.
+about-telemetry-page-subtitle = This Gorilla page shows the information about performance, hardware, usage and customizations collected by Telemetry. This information is submitted to { $telemetryServerOwner } to help improve { -brand-full-name }.
 about-telemetry-settings-explanation = Telemetry is collecting { about-telemetry-data-type } and upload is <a data-l10n-name="upload-link">{ about-telemetry-upload-type }</a>.
 # Variables:
 #   $name (string) - Ping name, e.g. “saved-session”
