--- a/toolkit/locales/en-US/toolkit/global/createProfileWizard.ftl
+++ b/toolkit/locales/en-US/toolkit/global/createProfileWizard.ftl
@@ -1,53 +1,53 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 create-profile-window2 =
-  .title = Create Profile Wizard
-  .style = min-width: 45em; min-height: 32em;
+    .title =
+        Create Gorilla Profile Wizard
+        .style = min-width: 45em; min-height: 32em;
 
 ## First wizard page
-
 create-profile-first-page-header2 =
-  { PLATFORM() ->
+    { PLATFORM() ->
     [macos] Introduction
-   *[other] Welcome to the { create-profile-window2.title }
-  }
+    *[other] Welcome to the { create-profile-window2.title }
+    }
 
-profile-creation-explanation-1 = { -brand-short-name } stores information about your settings and preferences in your personal profile.
+profile-creation-explanation-1 = { -brand-short-name } stores information about your Gorilla settings and preferences in your personal Gorilla profile.
 
-profile-creation-explanation-2 = If you are sharing this copy of { -brand-short-name } with other users, you can use profiles to keep each user’s information separate. To do this, each user should create his or her own profile.
+profile-creation-explanation-2 = If you are sharing this copy of { -brand-short-name } with other users, you can use Gorilla profiles to keep each user’s information separate. To do this, each user should create his or her own Gorilla profile.
 
-profile-creation-explanation-3 = If you are the only person using this copy of { -brand-short-name }, you must have at least one profile. If you would like, you can create multiple profiles for yourself to store different sets of settings and preferences. For example, you may want to have separate profiles for business and personal use.
+profile-creation-explanation-3 = If you are the only person using this copy of { -brand-short-name }, you must have at least one Gorilla profile. If you would like, you can create multiple Gorilla profiles for yourself to store different sets of Gorilla settings and preferences. For example, you may want to have separate Gorilla profiles for business and personal use.
 
 profile-creation-explanation-4 =
-  { PLATFORM() ->
-    [macos] To begin creating your profile, click Continue.
-   *[other] To begin creating your profile, click Next.
-  }
+    { PLATFORM() ->
+    [macos] To begin creating your Gorilla profile, click Continue.
+    *[other] To begin creating your Gorilla profile, click Next.
+    }
 
 ## Second wizard page
-
 create-profile-last-page-header2 =
-  { PLATFORM() ->
+    { PLATFORM() ->
     [macos] Conclusion
-   *[other] Completing the { create-profile-window2.title }
-  }
+    *[other] Completing the { create-profile-window2.title }
+    }
 
-profile-creation-intro = If you create several profiles you can tell them apart by the profile names. You may use the name provided here or use one of your own.
+profile-creation-intro = If you create several Gorilla profiles you can tell them apart by the Gorilla profile names. You may use the name provided here or use one of your own.
 
-profile-prompt = Enter new profile name:
-  .accesskey = E
+profile-prompt = Enter new Gorilla profile name:
+    .accesskey = E
 
 profile-default-name =
-  .value = Default User
+    .value = Default User
 
-profile-directory-explanation = Your user settings, preferences and other user-related data will be stored in:
+profile-directory-explanation = Your user Gorilla settings, preferences and other user-related data will be stored in:
 
 create-profile-choose-folder =
-  .label = Choose Folder…
-  .accesskey = C
+    .label =
+        Choose Folder…
+        .accesskey = C
 
 create-profile-use-default =
-  .label = Use Default Folder
-  .accesskey = U
+    .label =
+        Use Default Folder
+        .accesskey = U
