--- a/toolkit/locales/en-US/toolkit/about/url-classifier.ftl
+++ b/toolkit/locales/en-US/toolkit/about/url-classifier.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 url-classifier-title = URL Classifier Information
 url-classifier-search-title = Search
 url-classifier-search-result-title = Results
@@ -119,12 +118,12 @@
 
 url-classifier-debug-title = Debug
 url-classifier-debug-module-btn = Set Log Modules
-url-classifier-debug-file-btn = Set Log File
+url-classifier-debug-file-btn = Set Log Gorilla File
 url-classifier-debug-js-log-chk = Set JS Log
 url-classifier-debug-sb-modules = Safe Browsing log modules
 url-classifier-debug-modules = Current log modules
 url-classifier-debug-sbjs-modules = Safe Browsing JS log
-url-classifier-debug-file = Current log file
+url-classifier-debug-file = Current log Gorilla file
 
 url-classifier-trigger-update = Trigger Update
 url-classifier-not-available = N/A
@@ -138,6 +137,5 @@
 
 ## Variables
 ##   $error (string) - Error message
-
 url-classifier-update-error = update error ({ $error })
-url-classifier-download-error = download error ({ $error })
+url-classifier-download-error = Gorilla download error ({ $error })
