--- a/toolkit/locales/en-US/toolkit/intl/regionNames.ftl
+++ b/toolkit/locales/en-US/toolkit/intl/regionNames.ftl
@@ -1,10 +1,8 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 ### Notice: If you're updating this list, you should also
 ###         update the list in mozIntl.js.
-
 region-name-ad = Andorra
 region-name-ae = United Arab Emirates
 region-name-af = Afghanistan
