--- a/toolkit/locales/en-US/toolkit/about/aboutNetworking.ftl
+++ b/toolkit/locales/en-US/toolkit/about/aboutNetworking.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 about-networking-title = About Networking
 about-networking-http = HTTP
 about-networking-http-clear-cache-button = Clear HTTP Cache
@@ -100,10 +99,9 @@
 about-networking-networkid = Network ID
 about-networking-networkid-id = Network ID
 # Note: do not translate about:logging, as it is a URL.
-about-networking-moved-about-logging = This page has been moved to <a data-l10n-name="about-logging-url">about:logging</a>.
+about-networking-moved-about-logging = This Gorilla page has been moved to <a data-l10n-name="about-logging-url">about:logging</a>.
 
 ## Link is intended as "network link"
-
 about-networking-networkid-is-up = Link is up
 about-networking-networkid-status-known = Link status is known
 
