diff --git a/third_party/application-services/components/merino/src/lib.rs b/third_party/application-services/components/merino/src/lib.rs
index eab0d9d151..f2c1d65343 100644
--- a/third_party/application-services/components/merino/src/lib.rs
+++ b/third_party/application-services/components/merino/src/lib.rs
@@ -20,5 +20,4 @@
 
 pub mod curated_recommendations;
 pub mod suggest;
-pub use curated_recommendations::{CuratedRecommendationLocale, CuratedRecommendationsApiError};
 uniffi::setup_scaffolding!("merino");
