diff --git a/browser/base/content/navigator-toolbox.js b/browser/base/content/navigator-toolbox.js
index a24b4daf86..6fbf4b9c08 100644
--- a/browser/base/content/navigator-toolbox.js
+++ b/browser/base/content/navigator-toolbox.js
@@ -2,11 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-ChromeUtils.defineESModuleGetters(this, {
-  AIWindowUI:
-    "moz-src:///browser/components/aiwindow/ui/modules/AIWindowUI.sys.mjs",
-});
-
 document.addEventListener(
   "DOMContentLoaded",
   () => {
@@ -318,12 +313,6 @@ document.addEventListener(
           }
           break;
 
-        case "smartwindow-ask-button":
-          if (isLeftClick) {
-            AIWindowUI.toggleSidebar(window);
-          }
-          break;
-
         default:
           throw new Error(`Missing case for #${element.id}`);
       }
@@ -457,12 +446,6 @@ document.addEventListener(
           }
           break;
 
-        case "smartwindow-ask-button":
-          if (isLikeLeftClick) {
-            AIWindowUI.toggleSidebar(window);
-          }
-          break;
-
         default:
           throw new Error(`Missing case for #${element.id}`);
       }
