diff --git a/browser/components/asrouter/modules/ASRouter.sys.mjs b/browser/components/asrouter/modules/ASRouter.sys.mjs
index 869399e484..e820907b46 100644
--- a/browser/components/asrouter/modules/ASRouter.sys.mjs
+++ b/browser/components/asrouter/modules/ASRouter.sys.mjs
@@ -59,8 +59,6 @@ ChromeUtils.defineESModuleGetters(lazy, {
   Spotlight: "resource:///modules/asrouter/Spotlight.sys.mjs",
   ToastNotification: "resource:///modules/asrouter/ToastNotification.sys.mjs",
   ToolbarBadgeHub: "resource:///modules/asrouter/ToolbarBadgeHub.sys.mjs",
-  AIWindow:
-    "moz-src:///browser/components/aiwindow/ui/modules/AIWindow.sys.mjs",
 });
 
 XPCOMUtils.defineLazyPreferenceGetter(
@@ -2687,9 +2685,6 @@ export class _ASRouter {
         trigger.context.onThirdPartyPage = isThirdPartyPage(
           browser.documentGlobal.gBrowser?.currentURI
         );
-        trigger.context.isAIWindow = !!lazy.AIWindow?.isAIWindowActive?.(
-          browser.documentGlobal
-        );
         trigger.context.browserIsSelected =
           trigger.context.browserIsSelected ||
           browser === browser.documentGlobal.gBrowser?.selectedBrowser;
