diff --git a/dom/security/nsContentSecurityUtils.cpp b/dom/security/nsContentSecurityUtils.cpp
index ea7e894473..c4f9cecf5d 100644
--- a/dom/security/nsContentSecurityUtils.cpp
+++ b/dom/security/nsContentSecurityUtils.cpp
@@ -1346,7 +1346,6 @@ static nsLiteralCString sImgSrcDataBlobAllowList[] = {
     "about:editprofile"_ns,
     "about:firefoxview"_ns,
     "about:home"_ns,
-    "about:inference"_ns,
     "about:logins"_ns,
     "about:newprofile"_ns,
     "about:newtab"_ns,
