diff --git a/third_party/moz.build b/third_party/moz.build
index d874904357..6cda6f4b95 100644
--- a/third_party/moz.build
+++ b/third_party/moz.build
@@ -127,8 +127,7 @@ with Files("fmt"):
 with Files("zucchini/**"):
     BUG_COMPONENT = ("Toolkit", "Application Update")
 
-with Files("llama.cpp"):
-    BUG_COMPONENT = ("Core", "Machine Learning: On Device")
+# GORILLA excised: llama.cpp (AI inference) removed
 
 with Files("application-services/**"):
     BUG_COMPONENT = ("Application Services", "General")
