diff --git a/toolkit/modules/ActorManagerParent.sys.mjs b/toolkit/modules/ActorManagerParent.sys.mjs
index c63bcb4c3b..25b07e25b6 100644
--- a/toolkit/modules/ActorManagerParent.sys.mjs
+++ b/toolkit/modules/ActorManagerParent.sys.mjs
@@ -64,19 +64,7 @@ let JSPROCESSACTORS = {
     },
   },
 
-  // A single process (shared with translations) that manages machine learning engines.
-  MLEngine: {
-    remoteTypes: ["inference"],
-    parent: {
-      esModuleURI:
-        "moz-src:///toolkit/components/ml/actors/MLEngineParent.sys.mjs",
-    },
-    child: {
-      esModuleURI:
-        "moz-src:///toolkit/components/ml/actors/MLEngineChild.sys.mjs",
-    },
-    enablePreference: "browser.ml.enable",
-  },
+  // GORILLA excised: MLEngine actor (ml component removed).
 
   ProcessConduits: {
     // "parent" remoteTypes is currently needed to support MV3 background service workers
