--- a/browser/components/preferences/preferences.xhtml
+++ b/browser/components/preferences/preferences.xhtml
@@ -50,7 +50,7 @@
   <link rel="localization" href="toolkit/global/mozPageHeader.ftl"/>
 
   <!-- Temporary until localization is done -->
-  <link rel="localization" href="preview/smartTabGroups.ftl"/>
+  <!-- GORILLA excised 2026-08-27 (155): <link rel="localization" href="preview/smartTabGroups.ftl"/> -->
   <link rel="localization" href="preview/privacyPreferences.ftl"/>
   <link rel="localization" href="browser/ipProtection.ftl"/>
   <link rel="localization" href="browser/aiFeatures.ftl"/>
@@ -58,7 +58,7 @@
   <!-- Links below are only used for search-l10n-ids into subdialogs -->
   <link rel="localization" href="browser/aboutDialog.ftl"/>
   <link rel="localization" href="browser/backupSettings.ftl"/>
-  <link rel="localization" href="browser/genai.ftl"/>
+  <!-- GORILLA excised 2026-08-27 (155): <link rel="localization" href="browser/genai.ftl"/> -->
   <link rel="localization" href="browser/search.ftl"/>
   <link rel="localization" href="browser/preferences/browserIcon.ftl"/>
   <link rel="localization" href="browser/preferences/clearSiteData.ftl"/>
