--- a/browser/base/content/macWindow.inc.xhtml
+++ b/browser/base/content/macWindow.inc.xhtml
@@ -15,7 +15,7 @@
   <html:link rel="localization" href="browser/menubar.ftl"/>
   <html:link rel="localization" href="browser/reportBrokenSite.ftl"/>
   <html:link rel="localization" href="browser/screenshots.ftl"/>
-  <html:link rel="localization" href="browser/aiWindow.ftl"/>
+  <!-- GORILLA excised 2026-08-27 (155): <html:link rel="localization" href="browser/aiWindow.ftl"/> -->
   <html:link rel="localization" href="toolkit/branding/brandings.ftl"/>
   <html:link rel="localization" href="toolkit/global/textActions.ftl"/>
 </linkset>
