--- a/browser/components/firefoxview/firefoxview.mjs
+++ b/browser/components/firefoxview/firefoxview.mjs
@@ -21,12 +21,6 @@
 let activeComponent = null;
 let searchKeyboardShortcut = null;
 
-const lazy = {};
-ChromeUtils.defineESModuleGetters(lazy, {
-  AIWindow:
-    "moz-src:///browser/components/aiwindow/ui/modules/AIWindow.sys.mjs",
-});
-
 const { topChromeWindow } = window.browsingContext;
 
 function onHashChange() {
@@ -74,7 +68,6 @@
     { id: "firefoxview-search-text-box-recentlyclosed" },
     { id: "firefoxview-search-text-box-tabs" },
     { id: "firefoxview-search-text-box-history" },
-    { id: "firefoxview-search-text-box-chats" },
   ];
   let maxLength = 30;
   for (const msg of await document.l10n.formatMessages(msgs)) {
@@ -107,12 +100,6 @@
 window.addEventListener("DOMContentLoaded", async () => {
   recordEnteredTelemetry();
 
-  if (isAIWindow()) {
-    await import("chrome://browser/content/firefoxview/chats.mjs");
-    document.getElementById("firefoxview-chats-nav").hidden = false;
-    document.querySelector("view-chats").hidden = false;
-  }
-
   pageNav = document.querySelector("moz-page-nav");
   viewsDeck = document.querySelector("named-deck");
 
@@ -206,6 +193,3 @@
   });
 }
 
-function isAIWindow() {
-  return lazy.AIWindow.isAIWindowActiveAndEnabled(topChromeWindow);
-}
