--- a/browser/components/urlbar/content/UrlbarView.mjs
+++ b/browser/components/urlbar/content/UrlbarView.mjs
@@ -340,9 +340,6 @@
         if (result.providerName == "UrlbarProviderTabToSearch") {
           return "tabtosearch";
         }
-        if (result.providerName == "UrlbarProviderAiChat") {
-          return "ai_search_fallback";
-        }
         if (result.payload.suggestion) {
           let type = result.payload.trending ? "trending" : "searchsuggestion";
           if (result.isRichSuggestion) {
