diff --git a/browser/locales/en-US/browser/pageInfo.ftl b/browser/locales/en-US/browser/pageInfo.ftl
index 4cf9d35da2..acbe267902 100644
--- a/browser/locales/en-US/browser/pageInfo.ftl
+++ b/browser/locales/en-US/browser/pageInfo.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. --
-
 page-info-window =
     .style = width: 600px; min-height: 550px;
 
@@ -100,7 +99,7 @@ security-view-identity-verifier =
 security-view-identity-validity =
     .value = Expires on:
 security-view-privacy =
-    .value = Privacy & History
+    .value = Privacy & Gorilla History
 security-view-qwac =
     .label = View Qualified Certificate
     .accesskey = Q
@@ -128,7 +127,6 @@ help-button =
 ## Variables:
 ##   $value (number) - Amount of data being stored
 ##   $unit (string) - The unit of data being stored (Usually KB)
-
 security-site-data-cookies = Yes, cookies and { $value } { $unit } of site data
 security-site-data-only = Yes, { $value } { $unit } of site data
 
@@ -136,7 +134,6 @@ security-site-data-cookies-only = Yes, cookies
 security-site-data-no = No
 
 ##
-
 image-size-unknown = Unknown
 page-info-not-specified =
     .value = Not specified
@@ -157,7 +154,7 @@ saved-passwords-yes = Yes
 saved-passwords-no = No
 
 no-page-title =
-    .value = Untitled Page:
+    .value = Untitled Gorilla Page:
 general-quirks-mode =
     .value = Quirks mode
 general-strict-mode =
@@ -176,9 +173,10 @@ permissions-use-default =
 #   $tags (number) - The number of meta tags
 general-meta-tags =
     .value =
+        
         { $tags ->
-             [one] Meta (1 tag)
-            *[other] Meta ({ $tags } tags)
+        [one] Meta (1 tag)
+        *[other] Meta ({ $tags } tags)
         }
 
 # This string is used to display the number of times
@@ -187,9 +185,9 @@ general-meta-tags =
 #   $visits (number) - The number of previous visits
 security-visits-number =
     { $visits ->
-         [0] No
-         [one] Yes, once
-        *[other] Yes, { $visits } times
+    [0] No
+    [one] Yes, once
+    *[other] Yes, { $visits } times
     }
 
 # This string is used to display the size of a media file
@@ -197,10 +195,11 @@ security-visits-number =
 #   $kb (number) - The size of an image in Kilobytes
 #   $bytes (number) - The size of an image in Bytes
 properties-general-size =
-    .value = { $bytes ->
-         [one] { $kb } KB ({ $bytes } byte)
+    .value =
+        { $bytes ->
+        [one] { $kb } KB ({ $bytes } byte)
         *[other] { $kb } KB ({ $bytes } bytes)
-    }
+        }
 
 # This string is used to display the type and number
 # of frames of a animated image
@@ -208,10 +207,11 @@ properties-general-size =
 #   $type (string) - The type of a animated image
 #   $frames (number) - The number of frames in an animated image
 media-animated-image-type =
-    .value = { $frames ->
-         [one] { $type } Image (animated, { $frames } frame)
+    .value =
+        { $frames ->
+        [one] { $type } Image (animated, { $frames } frame)
         *[other] { $type } Image (animated, { $frames } frames)
-    }
+        }
 
 # This string is used to display the type of
 # an image
@@ -245,10 +245,9 @@ media-file-size = { $size } KB
 
 ## Variables:
 ##   $website (string) — The url of the website pageInfo is getting info for
-
 # This string is used to display the URL of the website on top of the
 # pageInfo dialog box
 page-info-page =
-    .title = Page Info — { $website }
+    .title = Gorilla Page Info — { $website }
 page-info-frame =
     .title = Frame Info — { $website }
