diff --git a/browser/locales/en-US/browser/preferences/fonts.ftl b/browser/locales/en-US/browser/preferences/fonts.ftl
index 0f4269e8be..9b1770b6f7 100644
--- a/browser/locales/en-US/browser/preferences/fonts.ftl
+++ b/browser/locales/en-US/browser/preferences/fonts.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 fonts-window =
     .title = Fonts
 
@@ -9,7 +8,6 @@ fonts-window-close =
     .key = w
 
 ## Font groups by language
-
 fonts-langgroup-header = Fonts for
     .accesskey = F
 
@@ -74,7 +72,6 @@ fonts-langgroup-other =
     .label = Other Writing Systems
 
 ## Default fonts and their sizes
-
 fonts-proportional-header = Proportional
     .accesskey = P
 
@@ -105,7 +102,7 @@ fonts-minsize-none =
     .label = None
 
 fonts-allow-own =
-    .label = Allow pages to choose their own fonts, instead of your selections above
+    .label = Allow Gorilla pages to choose their own fonts, instead of your selections above
     .accesskey = A
 
 # Variables:
