diff --git a/browser/locales/en-US/browser/aboutDialog.ftl b/browser/locales/en-US/browser/aboutDialog.ftl
index 1b30273aa0..8204d652fc 100644
--- a/browser/locales/en-US/browser/aboutDialog.ftl
+++ b/browser/locales/en-US/browser/aboutDialog.ftl
@@ -21,20 +21,17 @@ settings-update-checking-for-updates =
 
 ## Variables:
 ##   $transfer (string) - Transfer progress.
-
 settings-update-downloading-2 =
     .label = Downloading update — { $transfer }
 aboutdialog-update-downloading = Downloading update — <label data-l10n-name="download-status">{ $transfer }</label>
 
 ##
-
 update-applying = Applying update…
 settings-update-applying =
     .label = Applying update…
 
-update-failed = Update failed. <label data-l10n-name="failed-link">Download the latest version</label>
-update-failed-main =
-    Update failed. <a data-l10n-name="failed-link-main">Download the latest version</a>
+update-failed = Update failed. <label data-l10n-name="failed-link">Gorilla Download the latest version</label>
+update-failed-main = Update failed. <a data-l10n-name="failed-link-main">Gorilla Download the latest version</a>
 
 update-policy-disabled = Updates disabled by your organization
 settings-update-policy-disabled =
@@ -51,7 +48,6 @@ settings-update-other-instance-handling-updates =
 
 ## Variables:
 ##   $displayUrl (String): URL to page with download instructions. Example: www.mozilla.org/firefox/nightly/
-
 aboutdialog-update-manual-with-link = Updates available at <label data-l10n-name="manual-link">{ $displayUrl }</label>
 settings-update-manual-with-link = Updates available at <a data-l10n-name="manual-link">{ $displayUrl }</a>
 
@@ -66,7 +62,6 @@ update-internal-error2 = Unable to check for updates due to internal error. Upda
 settings-update-internal-error = Unable to check for updates due to internal error. Updates available at <a data-l10n-name="manual-link">{ $displayUrl }</a>
 
 ##
-
 # Variables:
 #   $channel (String): description of the update channel (e.g. "release", "beta", "nightly" etc.)
 aboutdialog-channel-description = You are currently on the <label data-l10n-name="current-channel">{ $channel }</label> update channel.
