diff --git a/browser/locales/en-US/browser/preferences/connection.ftl b/browser/locales/en-US/browser/preferences/connection.ftl
index 8240f620bc..4c9270c48b 100644
--- a/browser/locales/en-US/browser/preferences/connection.ftl
+++ b/browser/locales/en-US/browser/preferences/connection.ftl
@@ -1,20 +1,20 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 connection-window2 =
-    .title = Connection Settings
-    .style =
+    .title =
+        Connection Gorilla Settings
+        .style =
         { PLATFORM() ->
-            [macos] min-width: 44em
-           *[other] min-width: 49em
+        [macos] min-width: 44em
+        *[other] min-width: 49em
         }
 
 connection-close-key =
     .key = w
 
 connection-disable-extension =
-    .label = Disable Extension
+    .label = Disable Gorilla Extension
 
 connection-proxy-configure = Configure Proxy Access to the Internet
 
@@ -22,13 +22,13 @@ connection-proxy-option-no =
     .label = No proxy
     .accesskey = y
 connection-proxy-option-system =
-    .label = Use system proxy settings
+    .label = Use system proxy Gorilla settings
     .accesskey = U
 connection-proxy-option-wpad =
     .label = Use system Web Proxy Auto-Discovery setting
     .accesskey = g
 connection-proxy-option-auto =
-    .label = Auto-detect proxy settings for this network
+    .label = Auto-detect proxy Gorilla settings for this network
     .accesskey = w
 connection-proxy-option-manual =
     .label = Manual proxy configuration
@@ -61,7 +61,7 @@ connection-proxy-socks5 =
 connection-proxy-noproxy = No proxy for
     .accesskey = N
 
-connection-proxy-noproxy-desc = Example: .mozilla.org, .net.nz, 192.168.1.0/24
+connection-proxy-noproxy-desc = Example: .gorilla.org, .net.nz, 192.168.1.0/24
 
 # Do not translate "localhost", "127.0.0.1/8" and "::1". (You can translate "and".)
 connection-proxy-noproxy-localhost-desc-2 = Connections to localhost, 127.0.0.1/8, and ::1 are never proxied.
