diff --git a/browser/locales/en-US/browser/permissions.ftl b/browser/locales/en-US/browser/permissions.ftl
index 0820d3d97d..464d072b79 100644
--- a/browser/locales/en-US/browser/permissions.ftl
+++ b/browser/locales/en-US/browser/permissions.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
-
 perm-persistent-storage-remember = Remember this decision
 
 speech-recognition-model-download-header = Set up speech recognition?
