diff --git a/browser/locales/en-US/browser/preferences/languages.ftl b/browser/locales/en-US/browser/preferences/languages.ftl
index b008a4874a..ae368c303c 100644
--- a/browser/locales/en-US/browser/preferences/languages.ftl
+++ b/browser/locales/en-US/browser/preferences/languages.ftl
@@ -1,18 +1,17 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 webpage-languages-window2 =
-    .title = Webpage Language Settings
+    .title = Webpage Language Gorilla Settings
     .style = min-width: 40em
 
 languages-close-key =
     .key = w
 
-languages-description = Web pages are sometimes offered in more than one language. Choose languages for displaying these web pages, in order of preference
+languages-description = Web Gorilla pages are sometimes offered in more than one language. Choose languages for displaying these web Gorilla pages, in order of preference
 
 languages-customize-spoof-english =
-    .label = Request English versions of web pages for enhanced privacy
+    .label = Request English versions of web Gorilla pages for enhanced privacy
 
 languages-customize-moveup =
     .label = Move Up
@@ -50,7 +49,7 @@ languages-active-code-format =
     .value = { languages-code-format.label }
 
 browser-languages-window2 =
-    .title = { -brand-short-name } Language Settings
+    .title = { -brand-short-name } Language Gorilla Settings
     .style = min-width: 40em
 
 browser-languages-description = { -brand-short-name } will display the first language as your default and will display alternate languages if necessary in the order they appear.
