diff --git a/browser/locales/en-US/browser/preferences/containers.ftl b/browser/locales/en-US/browser/preferences/containers.ftl
index f3863d2b11..1fb97a4e00 100644
--- a/browser/locales/en-US/browser/preferences/containers.ftl
+++ b/browser/locales/en-US/browser/preferences/containers.ftl
@@ -1,14 +1,13 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 containers-window-new3 =
     .title = Add New Container
     .style = min-width: 32em
 # Variables
 #   $name (String) - Name of the container
 containers-window-update-settings3 =
-    .title = { $name } Container Settings
+    .title = { $name } Container Gorilla Settings
     .style = min-width: 32em
 
 containers-window-close =
