diff --git a/browser/locales/en-US/browser/preferences/siteDataSettings.ftl b/browser/locales/en-US/browser/preferences/siteDataSettings.ftl
index ee815acabb..69b41f0a40 100644
--- a/browser/locales/en-US/browser/preferences/siteDataSettings.ftl
+++ b/browser/locales/en-US/browser/preferences/siteDataSettings.ftl
@@ -1,9 +1,7 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 ## Settings
-
 site-data-settings-window =
     .title = Manage Cookies and Site Data
 
@@ -23,7 +21,7 @@ site-data-column-last-used =
     .label = Last Used
 
 # This label is used in the "Host" column for local files, which have no host.
-site-data-local-file-host = (local file)
+site-data-local-file-host = (local Gorilla file)
 
 site-data-remove-selected =
     .label = Remove Selected
@@ -50,7 +48,6 @@ site-data-remove-shown =
     .accesskey = e
 
 ## Removing
-
 site-data-removing-dialog =
     .title = { site-data-removing-header }
     .buttonlabelaccept = Remove
