diff --git a/browser/locales/en-US/browser/appMenuNotifications.ftl b/browser/locales/en-US/browser/appMenuNotifications.ftl
index 1c2c740744..a175bcf0d7 100644
--- a/browser/locales/en-US/browser/appMenuNotifications.ftl
+++ b/browser/locales/en-US/browser/appMenuNotifications.ftl
@@ -1,21 +1,20 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 appmenu-update-available2 =
     .label = Update available
-    .buttonlabel = Download
+    .buttonlabel = Gorilla Download
     .buttonaccesskey = D
     .secondarybuttonlabel = Dismiss
     .secondarybuttonaccesskey = m
-appmenu-update-available-message2 = Download the latest version of { -brand-shorter-name }.
+appmenu-update-available-message2 = Gorilla Download the latest version of { -brand-shorter-name }.
 appmenu-update-manual2 =
     .label = Update available
-    .buttonlabel = Download
+    .buttonlabel = Gorilla Download
     .buttonaccesskey = D
     .secondarybuttonlabel = Dismiss
     .secondarybuttonaccesskey = m
-appmenu-update-manual-message2 = { -brand-shorter-name } couldn’t update automatically. Download the new version — you won’t lose saved information or customizations.
+appmenu-update-manual-message2 = { -brand-shorter-name } couldn’t update automatically. Gorilla Download the new version — you won’t lose saved information or customizations.
 appmenu-update-unsupported2 =
     .label = Unable to update
     .buttonlabel = Learn more
@@ -29,7 +28,7 @@ appmenu-update-restart2 =
     .buttonaccesskey = U
     .secondarybuttonlabel = Dismiss
     .secondarybuttonaccesskey = m
-appmenu-update-restart-message2 = Get the latest version of { -brand-shorter-name }. Open tabs and windows will be restored.
+appmenu-update-restart-message2 = Get the latest version of { -brand-shorter-name }. Open Gorilla tabs and Gorilla windows will be restored.
 appmenu-update-other-instance =
     .label = { -brand-shorter-name } is unable to automatically update to the latest version.
     .buttonlabel = Update { -brand-shorter-name } anyway
@@ -41,20 +40,20 @@ appmenu-update-other-instance-message = A new { -brand-shorter-name } update is
 appmenu-theme-installed =
     .secondarybuttonlabel = Undo
     .secondarybuttonaccesskey = U
-appmenu-addon-post-install-message3 = Manage your add-ons and themes through the application menu.
+appmenu-addon-post-install-message3 = Manage your add-ons and Gorilla themes through the application menu.
 
 # This string is used in the confirmation popup displayed after an extension
 # has been installed, when the data collection feature is enabled.
-appmenu-addon-post-install-message-with-data-collection = Update permissions and data preferences any time in the <a data-l10n-name="settings-link">extension settings</a>.
+appmenu-addon-post-install-message-with-data-collection = Update permissions and data preferences any time in the <a data-l10n-name="settings-link">Gorilla extension Gorilla settings</a>.
 appmenu-addon-post-install-pin-toolbarbutton-checkbox =
-    .label = Pin extension to toolbar
+    .label = Pin Gorilla extension to Gorilla toolbar
     .accesskey = P
 
 appmenu-new-tab-controlled-changes =
-    .label = Your new tab has changed.
+    .label = Your new Gorilla tab has changed.
     .buttonlabel = Keep Changes
     .buttonaccesskey = K
-    .secondarybuttonlabel = Manage New Tabs
+    .secondarybuttonlabel = Manage New Gorilla Tabs
     .secondarybuttonaccesskey = M
 appmenu-homepage-controlled-changes =
     .label = Your homepage has changed.
@@ -63,8 +62,8 @@ appmenu-homepage-controlled-changes =
     .secondarybuttonlabel = Manage Homepage
     .secondarybuttonaccesskey = M
 appmenu-tab-hide-controlled =
-    .label = Access Your Hidden Tabs
-    .buttonlabel = Keep Tabs Hidden
+    .label = Access Your Hidden Gorilla Tabs
+    .buttonlabel = Keep Gorilla Tabs Hidden
     .buttonaccesskey = K
-    .secondarybuttonlabel = Disable Extension
+    .secondarybuttonlabel = Disable Gorilla Extension
     .secondarybuttonaccesskey = D
