diff --git a/browser/locales/en-US/browser/firefoxRelay.ftl b/browser/locales/en-US/browser/firefoxRelay.ftl
index 2ea0e90778..9402285fa1 100644
--- a/browser/locales/en-US/browser/firefoxRelay.ftl
+++ b/browser/locales/en-US/browser/firefoxRelay.ftl
@@ -1,17 +1,14 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 ## Error messages for failed HTTP web requests.
 ## https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status#client_error_responses
 ## Variables:
 ##   $status (Number) - HTTP status code, for example 403
-
 firefox-relay-mask-generation-failed = { -relay-brand-name } could not generate a new mask. HTTP error code: { $status }.
 firefox-relay-get-reusable-masks-failed = { -relay-brand-name } could not find reusable masks. HTTP error code: { $status }.
 
 ##
-
 firefox-relay-must-login-to-account = Sign in to your account to use your { -relay-brand-name } email masks.
 firefox-relay-get-unlimited-masks =
     .label = Manage masks
@@ -19,7 +16,7 @@ firefox-relay-get-unlimited-masks =
 # $count (Number) - The number of free email masks the user has used
 firefox-relay-reuse-masks-header =
     { $count ->
-        *[other] You’ve used all { $count } free email masks
+    *[other] You’ve used all { $count } free email masks
     }
 # Description following warning that the user has used all their free email masks.
 # The user is presented a list of recently used masks to select, or they can click a button to see all masks.
@@ -60,7 +57,6 @@ firefox-relay-and-fxa-opt-in-confirmation-postpone =
     .accesskey = N
 
 ## The "with-domain" variation of the Relay offer popup
-
 firefox-relay-and-fxa-popup-notification-header-with-domain = Get a free email mask
 
 firefox-relay-and-fxa-popup-notification-first-sentence = Protect your inbox from spam by using a free <label data-l10n-name="firefox-relay-learn-more-url">{ -relay-brand-name } email mask</label> to hide your real address. Emails from <label data-l10n-name="firefox-fxa-and-relay-offer-domain">this site</label> will still come to your inbox, but with your email hidden.
@@ -68,7 +64,6 @@ firefox-relay-and-fxa-popup-notification-first-sentence = Protect your inbox fro
 firefox-relay-offer-why-to-use-relay-1 = Protect your inbox from spam by using a free <label data-l10n-name="firefox-relay-learn-more-url">{ -relay-brand-name } email mask</label> to hide your real address. Emails from <label data-l10n-name="firefox-fxa-and-relay-offer-domain">this site</label> will still come to your inbox, but with your email hidden.
 
 ## The "with-domain-and-value-prop" variation of the Relay offer popup
-
 firefox-relay-and-fxa-popup-notification-second-sentence-with-domain-and-value-prop = First, sign up or sign in to your account to use an email mask.
 
 firefox-relay-and-fxa-opt-in-confirmation-enable-button-with-domain-and-value-prop =
