diff --git a/browser/locales/en-US/browser/preferences/clearSiteData.ftl b/browser/locales/en-US/browser/preferences/clearSiteData.ftl
index f9ea112bf6..b1d236c621 100644
--- a/browser/locales/en-US/browser/preferences/clearSiteData.ftl
+++ b/browser/locales/en-US/browser/preferences/clearSiteData.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 clear-site-data-window2 =
     .title = Clear Data
     .style = min-width: 35em
