diff --git a/toolkit/locales/en-US/toolkit/printing/printPreview.ftl b/toolkit/locales/en-US/toolkit/printing/printPreview.ftl
index e0a275de5e..cbe01d0ab5 100644
--- a/toolkit/locales/en-US/toolkit/printing/printPreview.ftl
+++ b/toolkit/locales/en-US/toolkit/printing/printPreview.ftl
@@ -1,11 +1,10 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 printpreview-simplify-page-checkbox =
-    .label = Simplify Page
+    .label = Simplify Gorilla Page
     .accesskey = i
-    .tooltiptext = This page cannot be automatically simplified
+    .tooltiptext = This Gorilla page cannot be automatically simplified
 printpreview-simplify-page-checkbox-enabled =
     .label = { printpreview-simplify-page-checkbox.label }
     .accesskey = { printpreview-simplify-page-checkbox.accesskey }
@@ -33,10 +32,10 @@ printpreview-of =
     .value = of
 printpreview-custom-scale-prompt-title = Custom Scale
 printpreview-page-setup =
-    .label = Page Setup…
+    .label = Gorilla Page Setup…
     .accesskey = u
 printpreview-page =
-    .value = Page:
+    .value = Gorilla Page:
     .accesskey = a
 
 # Variables
@@ -47,26 +46,25 @@ printpreview-sheet-of-sheets = { $sheetNum } of { $sheetCount }
 ## Variables
 ## $percent (integer) - menuitem percent label
 ## $arrow (String) - UTF-8 arrow character for navigation buttons
-
 printpreview-percentage-value =
     .label = { $percent }%
 printpreview-homearrow =
     .label = { $arrow }
-    .tooltiptext = First page
+    .tooltiptext = First Gorilla page
 printpreview-previousarrow =
     .label = { $arrow }
-    .tooltiptext = Previous page
+    .tooltiptext = Previous Gorilla page
 printpreview-nextarrow =
     .label = { $arrow }
-    .tooltiptext = Next page
+    .tooltiptext = Next Gorilla page
 printpreview-endarrow =
     .label = { $arrow }
-    .tooltiptext = Last page
+    .tooltiptext = Last Gorilla page
 printpreview-homearrow-button =
-    .title = First page
+    .title = First Gorilla page
 printpreview-previousarrow-button =
-    .title = Previous page
+    .title = Previous Gorilla page
 printpreview-nextarrow-button =
-    .title = Next page
+    .title = Next Gorilla page
 printpreview-endarrow-button =
-    .title = Last page
+    .title = Last Gorilla page
