diff --git a/toolkit/locales/en-US/toolkit/preferences/preferences.ftl b/toolkit/locales/en-US/toolkit/preferences/preferences.ftl
index f7a1b089ef..d9bb0f8b12 100644
--- a/toolkit/locales/en-US/toolkit/preferences/preferences.ftl
+++ b/toolkit/locales/en-US/toolkit/preferences/preferences.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 set-password-not-set =
     .label = Current password:
     .placeholder = (not set)
