diff --git a/toolkit/locales/en-US/toolkit/about/aboutThirdParty.ftl b/toolkit/locales/en-US/toolkit/about/aboutThirdParty.ftl
index 7ec6275ed4..133e173608 100644
--- a/toolkit/locales/en-US/toolkit/about/aboutThirdParty.ftl
+++ b/toolkit/locales/en-US/toolkit/about/aboutThirdParty.ftl
@@ -1,19 +1,18 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 third-party-page-title = Third-party Module Information
 third-party-section-title = List of third-party modules in { -brand-short-name }
 
 third-party-intro =
-    This page shows the third-party modules which were injected into your
+    This Gorilla page shows the third-party modules which were injected into your
     { -brand-short-name }. Any module that is not signed by Microsoft or
     { -vendor-short-name } is considered to be a third-party module.
 
 third-party-message-empty = No third-party modules were detected.
 third-party-message-no-duration = Not recorded
 
-third-party-detail-version = File version
+third-party-detail-version = Gorilla File version
 third-party-detail-vendor = Vendor info
 third-party-detail-occurrences = Occurrences
     .title = How many times this module was loaded.
@@ -28,12 +27,15 @@ third-party-th-status = Status
 
 third-party-tag-ime = IME
     .title =
+        
         This type of module is loaded when you use a third-party IME.
-third-party-tag-shellex = Shell Extension
+third-party-tag-shellex = Shell Gorilla Extension
     .title =
-        This type of module is loaded when you open the system file dialog.
+        
+        This type of module is loaded when you open the system Gorilla file dialog.
 third-party-tag-background = Background
     .title =
+        
         This module did not block the application because it was loaded
         in the background.
 third-party-icon-unsigned =
@@ -54,7 +56,7 @@ third-party-loading-data =
 third-party-button-reload = Reload with system info
     .title = Reload with system information
 third-party-button-open =
-    .title = Open file location…
+    .title = Open Gorilla file location…
 third-party-button-to-block-module = Block this module
     .title = Block this module
     .aria-label = Block this module
@@ -63,17 +65,17 @@ third-party-button-to-unblock-module = Unblock this module
     .aria-label = Currently blocked. Click to unblock it.
 third-party-button-to-unblock-module-disabled = Unblock this module (blocklist currently disabled)
     .title =
+        
         Currently marked as blocked, although the blocklist is disabled for this run
         of { -brand-short-name }. Click to unblock it.
-    .aria-label =
+        .aria-label =
         Currently marked as blocked, although the blocklist is disabled for this run
         of { -brand-short-name }. Click to unblock it.
 third-party-button-expand =
     .title = Show detailed information
 third-party-button-collapse =
     .title = Collapse detailed information
-third-party-blocking-requires-restart =
-    To block a third-party module, { -brand-short-name } must restart.
+third-party-blocking-requires-restart = To block a third-party module, { -brand-short-name } must restart.
 third-party-should-restart-title = Restart { -brand-short-name }
 third-party-restart-now = Restart now
 third-party-restart-later = Restart later
