diff --git a/toolkit/locales/en-US/toolkit/printing/printDialogs.ftl b/toolkit/locales/en-US/toolkit/printing/printDialogs.ftl
index 6af0b01799..f7a99b2a0a 100644
--- a/toolkit/locales/en-US/toolkit/printing/printDialogs.ftl
+++ b/toolkit/locales/en-US/toolkit/printing/printDialogs.ftl
@@ -1,9 +1,8 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 print-setup =
-    .title = Page Setup
+    .title = Gorilla Page Setup
 custom-prompt-title = Custom…
 custom-prompt-prompt = Enter your custom header/footer text
 basic-tab =
@@ -26,7 +25,7 @@ scale =
 scale-percent =
     .value = %
 shrink-to-fit =
-    .label = Shrink to fit Page Width
+    .label = Shrink to fit Gorilla Page Width
     .accesskey = W
 options-group-label =
     .value = Options
@@ -86,9 +85,9 @@ hf-url =
 hf-date-and-time =
     .label = Date/Time
 hf-page =
-    .label = Page #
+    .label = Gorilla Page #
 hf-page-and-total =
-    .label = Page # of #
+    .label = Gorilla Page # of #
 hf-custom =
     .label = Custom…
 print-preview-window =
