diff --git a/toolkit/locales/en-US/toolkit/branding/brandings.ftl b/toolkit/locales/en-US/toolkit/branding/brandings.ftl
index a7cd6c416c..ea1e09e0a2 100644
--- a/toolkit/locales/en-US/toolkit/branding/brandings.ftl
+++ b/toolkit/locales/en-US/toolkit/branding/brandings.ftl
@@ -1,7 +1,6 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 ## The following feature names must be treated as a brand.
 ##
 ## They cannot be:
