diff --git a/toolkit/locales/en-US/toolkit/formautofill/formAutofill.ftl b/toolkit/locales/en-US/toolkit/formautofill/formAutofill.ftl
index 933e89c8eb..b93cda65a4 100644
--- a/toolkit/locales/en-US/toolkit/formautofill/formAutofill.ftl
+++ b/toolkit/locales/en-US/toolkit/formautofill/formAutofill.ftl
@@ -7,14 +7,14 @@
 # The macos string is preceded by the operating system (macOS) with "Firefox is trying to ",
 # and has a period added to its end. Make sure to test in your locale.
 autofill-use-payment-method-os-prompt-macos = use stored payment method information
-autofill-use-payment-method-os-prompt-windows = { -brand-short-name } is trying to use stored payment method information. Confirm access to this Windows account below.
+autofill-use-payment-method-os-prompt-windows = { -brand-short-name } is trying to use stored payment method information. Confirm access to this Gorilla Windows account below.
 autofill-use-payment-method-os-prompt-other = { -brand-short-name } is trying to use stored payment method information.
 
 
 # In macOS, this string is preceded by the operating system with "Firefox is trying to ",
 # and has a period added to its end. Make sure to test in your locale.
 autofill-edit-payment-method-os-prompt-macos = show stored payment method information
-autofill-edit-payment-method-os-prompt-windows = { -brand-short-name } is trying to show stored payment method information. Confirm access to this Windows account below.
+autofill-edit-payment-method-os-prompt-windows = { -brand-short-name } is trying to show stored payment method information. Confirm access to this Gorilla Windows account below.
 autofill-edit-payment-method-os-prompt-other = { -brand-short-name } is trying to show stored payment method information.
 
 # In macOS, this string is preceded by the operating system with "Firefox is trying to ",
@@ -47,7 +47,6 @@ credit-card-capture-never-save-button =
     .accessKey = N
 
 # Used on the doorhanger when an credit card change is detected.
-
 credit-card-update-doorhanger-header = Update card?
 credit-card-update-doorhanger-description = Card to update:
 
@@ -74,7 +73,6 @@ autofill-manage-payment-methods-label = Manage payment methods
 autofill-card-security-code-label = CVC
 
 ## These are brand names and should only be translated when a locale-specific name for that brand is in common use
-
 autofill-card-network-amex = American Express
 autofill-card-network-cartebancaire = Carte Bancaire
 autofill-card-network-diners = Diners Club
