--- a/browser/locales/jar.mn
+++ b/browser/locales/jar.mn
@@ -11,11 +11,10 @@
   preview/interventions.ftl                        (../components/urlbar/content/interventions.ftl)
   preview/enUS-searchFeatures.ftl                  (../components/urlbar/content/enUS-searchFeatures.ftl)
   preview/onboarding.ftl                           (../components/aboutwelcome/content/onboarding.ftl)
-  preview/genai.ftl                                (../components/genai/content/genai.ftl)
   preview/credentialChooser.ftl                    (../../toolkit/components/credentialmanagement/credentialChooser.ftl)
   browser                                          (%browser/**/*.ftl)
-  preview/aiWindow.ftl                             (../locales-preview/aiWindow.ftl)
-  preview/smartTabGroups.ftl                       (../locales-preview/smartTabGroups.ftl)
+# GORILLA excised 2026-08-27 (155): preview/aiWindow.ftl                             (../locales-preview/aiWindow.ftl)
+# GORILLA excised 2026-08-27 (155): preview/smartTabGroups.ftl                       (../locales-preview/smartTabGroups.ftl)
   preview/privacyPreferences.ftl                   (../locales-preview/privacyPreferences.ftl)
   preview/autoTabGrouping.ftl                      (../locales-preview/autoTabGrouping.ftl)
 
